Sample Data/Custom/TaniumThreatResponse_CL.csv

```diff
@@ -0,0 +1,310 @@
+﻿TenantId,SourceSystem,MG,ManagementGroupName,"TimeGenerated [UTC]",Computer,RawData,"Impact_Score","Match_Details_match_properties_parent_parent","Match_Details_finding_artifact_artifact_hash","Match_Details_finding_artifact_instance_hash","Match_Details_finding_artifact_windows_defender_event_event","Match_Details_finding_artifact_windows_defender_event_timestamp_ms","Match_Details_finding_description","Match_Details_finding_first_seen_t [UTC]","Match_Details_finding_hunt_id","Match_Details_finding_intel_id","Match_Details_finding_last_seen_t [UTC]","Match_Details_finding_source_name","Match_Details_finding_threat_id","Match_Details_finding_whats","Match_Details_match_properties_file_md5","Match_Details_match_properties_md5","Match_Details_match_properties_parent_start_time_t [UTC]","Match_Details_match_properties_start_time_t [UTC]","Timestamp","Alert_Id","Computer_IP","Computer_Name","Impact_Score","Intel_Id","Intel_Labels","Intel_Name","Intel_Type","MITRE_Techniques","Match_Details_finding_system_info_bits","Match_Details_finding_system_info_build_number","Match_Details_finding_system_info_os","Match_Details_finding_system_info_patch_level","Match_Details_finding_system_info_platform","Match_Details_hash","Match_Details_match_contexts","Match_Details_match_hash","Match_Details_match_properties_args","Match_Details_match_properties_cwd","Match_Details_match_properties_file_fullpath","Match_Details_match_properties_file_md5","Match_Details_match_properties_fullpath","Match_Details_match_properties_md5","Match_Details_match_properties_name","Match_Details_match_properties_parent_args","Match_Details_match_properties_parent_file","Match_Details_match_properties_parent_name","Match_Details_match_properties_parent_pid","Match_Details_match_properties_parent_ppid","Match_Details_match_properties_parent_recorder_unique_id","Match_Details_match_properties_parent_start_time","Match_Details_match_properties_parent_user","Match_Details_match_properties_pid","Match_Details_match_properties_ppid","Match_Details_match_properties_recorder_unique_id","Match_Details_match_properties_sha1","Match_Details_match_properties_sha256","Match_Details_match_properties_size","Match_Details_match_properties_start_time","Match_Details_match_properties_user","Match_Details_match_source","Match_Details_match_type","Match_Details_match_version","Match_Details_service_id","Timestamp_t [UTC]","fake",Type,"_ResourceId"
+"dummyTenantID",RestAPI,,,"6/20/2022, 9:58:45.405 AM",,,,"{
```

Comment:
Hey @ian-hands-tanium, Impact_Score, Match_Details_match_properties_file_md5, Match_Details_match_properties_md5 these columns/fields are added 2 times with same name, will you please the data by removing them from one instance if not needed, thanks.